api/definitions/definitions.proto

```diff
@@ -0,0 +1,895 @@
+syntax = "proto3";
+
+package definitions;
```

Comment:
I think proto files should mirror as much as possible structure of the resources, so that messages has more meaningful names, e.g. `cluster.proto` with `cluster/*` resources? and there should be a package `talos.cluster`, so that structure name is `talos.cluster.AffiliateSpec` for example

Reply:
Splitting files is okay, but dependencies is the thing I really don't want to implement, since it would require us to implement dependency graph. Are were sure we want it?

Reply:
A since proto doesn't handle cyclic dependencies you can't just generate one stab and reuse it everywhere.

Reply:
I don't see how there can be any cyclic dependencies between resources in different packages. Can you provide an example?